package.json

```diff
@@ -6,8 +6,7 @@
   "module": "dist/transformer.mjs",
   "exports": {
     "import": "./dist/transformer.mjs",
-    "require": "./dist/transformer.cjs",
-    "./package.json": "./package.json"
```

Comment:
some plugins like `rollup-plugin-svelte` will log a warning if `package.json` is not exported, so I added it back. I updated the default export, so if you were having any issue hopefully that will solve it

Reply:
Okay, vscode gave me an error on this line.